kover-cli/build.gradle.kts

```diff
@@ -23,7 +23,6 @@ plugins {
 
 extensions.configure<Kover_publishing_conventions_gradle.KoverPublicationExtension> {
     description.set("Command Line Interface for Kotlin Coverage Toolchain")
-    fatJar.set(true)
 }
```

Comment:
Why CLI shouldn't be a fat jar anymore? Will users still be able to do `java -jar kover-cli.jar ...`?

Reply:
I just don't see it in this buildscript. Is it added by default by conventions?

Reply:
Yes, applying fat jar convention plugin allows you to create archives if you use the `fatJarConfiguration`